test/functional/test_site/expected/sub_site/testReuse._include_.html

```diff
@@ -0,0 +1,18 @@
+<p>This is a page from another Markbind site. The purpose of this page is to ensure that reuse works as expected. All the following images should display correctly.</p>
+<p>Some variables:</p>
+<div></div>
+<p>IMG tags:
+  <img src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">
+  <img src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">
+  <img src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">
+  <img src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">
+  <img src="https://dummyimage.com/600x400/000/fff"></p>
+<p>PIC tags:
+  <pic src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png"></pic>
+  <pic src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png"></pic>
```

Comment:
When running `markbind serve` on the `test_site` folder and replacing the `_site` contents into `expected` folder

I encountered this:
![path diff](https://user-images.githubusercontent.com/31084833/54101973-5f894b00-4401-11e9-804d-a45c8b73092a.PNG)

It works fine when viewing in the browser though.

Reply:
Nice catch! The paths need to be further processed with `ensurePosix()`, so that on Windows, the path separator will consistently be `/`.

Could have caught this via CI if #448 is implemented. :P

Reply:
Nice catch! Thanks @Chng-Zhi-Xuan .